content/dstu2/allergy-intolerance.md

```diff
@@ -19,15 +19,31 @@ Search for AllergyIntolerances that meet supplied query parameters:
 
 ### Parameters
 
- Name      | Required? | Type          | Description
------------|-----------|---------------|-----------------------------------------------------------------
- `patient` | Y         | [`reference`] | Who the sensitivity is for. Example: `12345`
- `status`  | N         | [`token`]     | [Certainty of the allergy or intolerance]. Example: `confirmed`
+ Name      | Required?                                                          | Type          | Description
+-----------|--------------------------------------------------------------------|---------------|-----------------------------------------------------------------------
+ `_id`     | This or patient, if populated all other parameters are not allowed | [`token`]     | The logical resource id associated with the resource. Example: `12345`
```

Comment:
use tick marks around patient so it's stylized (recognized as a field name) like the _id parameter for patient.